tools/prowgen/cmd/prowgen/main.go

```diff
@@ -44,7 +44,7 @@ var (
 	preprocessCommand   = flag.String("pre-process-command", "", "command to run to preprocess the meta config files")
 	postprocessCommand  = flag.String("post-process-command", "", "command to run to postprocess the generated config files")
 	longJobNamesAllowed = flag.Bool("allow-long-job-names", false, "allow job names that are longer than 63 characters")
-	skipGarTagging = flag.Bool("skip-gar-tagging", false, "skip tagging gar images since that is permitted by few folks")
+	skipGarTagging      = flag.Bool("skip-gar-tagging", false, "skip tagging gar images since that is permitted by few folks")
```

Comment:
do we need this?

Reply:
It's just my editor running gofmt again, this time it's minor and i don't think it hurts through, rest of the file is already formatted.